rootfs/registry/dockerclient.py

```diff
@@ -55,8 +54,8 @@ def publish_release(self, source, config, target, deis_registry):
     def build(self, source, config, repo, tag):
         """Add a "last-mile" layer of environment config to a Docker image for deis-registry."""
         check_blacklist(repo)
-        env = ' '.join("{}='{}'".format(
-            k, v.encode('unicode-escape').replace("'", "\\'")) for k, v in config.viewitems())
+        env = ' '.join("{}='{}'".format(k, v) for k, v in config.items())
```

Comment:
I was having issues in the test cases where I saw `b'DEIS'` inside the expected quotes, could be a function of the test framework tho